### PR TITLE
j2objc(Podspec|Xcode) disabled for translateOnlyMode

### PR DESCRIPTION
**Pull request by brunobowden**
- Necessary when running on Windows & Linux

### CODE REVIEW

**Comment by brunobowden** [comment]
@advayDev1 - I haven't tried this on Windows or Linux but since we moved j2objcXcode to part of the build, it must be possible to disable it on that platform.

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcConfig.groovy [hunk 1]
```diff
@@ -853,7 +853,8 @@ class J2objcConfig {
                 // First pattern matches all native-compilation tasks.
                 // Second pattern matches plugin-specific tasks beyond translation.
                 if ((name =~ /^.*((J|j)2objc(Executable|StaticLibrary|SharedLibrary|Objc))$/).matches() ||
-                    (name =~ /^j2objc(Assemble|PackLibraries|Test)(Debug|Release)$/).matches()) {
+                    (name =~ /^j2objc(Assemble|PackLibraries|Test)(Debug|Release)$/).matches() ||
+                    (name =~ /^j2objc(Podspec|Xcode)$/).matches()) {
                     task.enabled = false
```
Comment (advayDev1) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcConfig.groovy]: Can we add a project.logger.info("J2objcPlugin: Disabling $name in translateOnlyMode");

**Comment by advayDev1** [comment]
lgtm + comment

**Comment by brunobowden** [comment]
That'll spew out a lot of noise in the build. The user will already see `shared:j2objcXcode SKIPPED` anyway, so I don't feel this adds value. I've add a single info log that translateOnlyMode is enabled but that will only be shown once.

**Pull request merged**